prism/src/main/java/org/apache/falcon/jdbc/MonitoringJdbcStateStore.java

```diff
@@ -244,7 +244,7 @@ private void commitAndCloseTransaction(EntityManager entityManager) {
         entityManager.close();
     }
 
-    public EntitySLAAlertBean getEntityAlertInstance(String entityName, String clusterName, Date nominalTime,
+    public List<EntitySLAAlertBean> getAlertInstancesList(String entityName, String clusterName, Date nominalTime,
```

Comment:
Why is this a list? For the given set of params, there can be only one entry. The entry may have SLALow miss or SLAHigh miss. But, there should be only one entry. Right?